evennia/commands/default/building.py

```diff
@@ -515,12 +515,18 @@ class CmdDesc(MuxCommand):
     """
     describe an object
 
-    Usage:
+    Usage 1:
       @desc [<obj> =] >description>
 
-    Setts the "desc" attribute on an
+    Sets the "desc" attribute on an
     object. If an object is not given,
     describe the current room.
+
+    Usage 2:
+      @desc/long [obj]
```

Comment:
This seems to refer to "/long" still, whereas the switch is /edit in the code below.